<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMainWindowFileSystem</class>
 <widget class="QMainWindow" name="TMainWindowFileSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QListView" name="listView"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>设置根目录</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QTreeView" name="treeView"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>TMainWindowFileSystem</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
